cirq-core/cirq/__init__.py

```diff
@@ -30,9 +30,9 @@
     from cirq import (
         # Core
         protocols,
-        value,
         linalg,
         qis,
+        value,
```

Comment:
Why was this moved?

Reply:
One of the tests was failing due to a recursive dependency. Can't find which one, sorry. Reverted now.